data/db-schema/src/main/resources/net/corda/db/schema/vnode-vault/migration/ledger-utxo-visible-transaction-state-creation-v1.0.xml

```diff
@@ -3,7 +3,10 @@
                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
 
-    <changeSet author="R3.Corda" id="ledger-utxo-visible-transaction-state-creation-postgres-v1.0" dbms="postgresql">
+    <property name="json.column.type" value="JSONB" dbms="postgresql"/>
+    <property name="json.column.type" value="VARCHAR(1024)" dbms="hsqldb"/>
```

Comment:
@mnesbit @blsemo There was an argument about using `CLOB` instead of `VARCHAR(1024)`. Any strong opinions on this?

Reply:
Yes I have a strong opinion that our public API shouldn't suggest we support a DB type we haven't in fact made work

Reply:
Personally, I think we _should_ make HSQLDB work.

Reply:
> Yes I have a strong opinion that our public API shouldn't suggest we support a DB type we haven't in fact made work.

Happily, we can still add this value for HSQLDB _programmatically_ in `LiquibaseSchemaMigratorImpl`:
```kotlin
 lb.changeLogParameters.set("json.column.type", "CLOB", null, null, "hsqldb")
```
So this should make everyone happy.

Reply:
I'm happy to go either way, just wanted to make sure we are on the same page here. @mnesbit what do you think?

Reply:
I have chatted to Chris and I am ok with it too

Reply:
I've made the changes, @mnesbit @chrisr3 let me know what you think.